integration/test/API/Galley.hs

```diff
@@ -592,13 +592,6 @@ legalholdUserStatus tid ownerid user = do
   req <- baseRequest ownerid Galley Versioned (joinHttpPath ["teams", tidS, "legalhold", uid])
   submit "GET" req
 
--- | https://staging-nginz-https.zinfra.io/v5/api/swagger-ui/#/default/post_teams__tid__legalhold_settings
```

Comment:
i think it makes sense to keep all api client functions even if they're not used anywhere, but i'm not sure.